### ISSUE TITLE
jaro_winkler distance function name

### ISSUE THREAD

**Issue by J535D165** [jamesturk/jellyfish#72]
The distance function `jaro_winkler` has no `_distance` extension. Is it an idea to change it into `jaro_winkler_distance`? 

```
levenshtein_distance
damerau_levenshtein_distance
jaro_distance
jaro_winkler
hamming_distance
```

**Comment by nchammas** [comment]
Was just wondering the same myself. 😀

If we rename it, we'd probably want to keep the old name around and add a deprecation warning in there to smooth the transition for users. (Though technically, Jellyfish is still pre-1.0 so breaking changes like this should be kosher. Not pleasant for users though.)

**Comment by J535D165** [comment]
Yes, I can make a depreciation warning for Python.  Unfortunately, my C skills are not enough to fix the C version. Maybe @jamesturk can do that?

**Comment by brandonscript** [comment]
Came here for this 😆

**Comment by jamesturk** [comment]
fixed in upcoming 0.8
